src/pages/index.astro

```diff
@@ -57,3 +57,4 @@ import Header from "../components/common/Header.astro";
   <People />
   <Footer />
 </Layout>
+ ../components/common/CarouselMobile.tsx
```

Comment:
This is not needed.

Reply:
was thinking of having a different component.